mu-plugins/blocks/global-header-footer/src/footer/index.js

```diff
@@ -14,6 +14,7 @@ import metadata from './block.json';
 
 const variations = [
 	{ label: __( 'White on black (default)', 'wporg' ), value: 'white-on-black' },
+	{ label: __( 'White on dark black', 'wporg' ), value: 'white-on-dark-black' },
 	{ label: __( 'Black on white', 'wporg' ), value: 'black-on-white' },
 	{ label: __( 'White on blue', 'wporg' ), value: 'white-on-blue' },
```

Comment:
This reminds me of colour palettes I've seen where one ends up adding superlatives for different shades, eg. dark grey, darker grey, darkest grey, super dark grey, etc.

To avoid this I wonder if we should reference the actual name of the preset, ie. `white-on-charcoal-1`, `white-on-blueberry-1`. I realise there is an intended simplification here with abstracting away the underlying color, but we're already seeing expansion of the palette, and number based naming is designed to address this problem, as there is an infinite scale built in.

Reply:
I think that's a great idea. I was cringing a little when I was trying to think of a name, but sticking to the actual preset name would simplify things. I'm happy to update this PR if others agree.

Reply:
Agreed. When I first named these, I assumed this was the total set, so I went with more human-friendly names. But since we're adding more, using the actual color names would be better.